fastlane/Fastfile

```diff
@@ -55,21 +55,23 @@ platform :ios do
     crashlytics(
       crashlytics_path: './Pods/Crashlytics/',
       groups: options[:group],
-      notifications: 'true'
+      notifications: 'true',
+      notes: options[:notes]
```

Comment:
Pretty sure we want to not have the parameter at all but more investigation is required.